.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -4,13 +4,14 @@ about: Report a bug encountered with Kubeflow Pipelines
 
 ---
```

Comment:
How do you like the idea of calling out KFP version and SDK version? I vaguely remember for 90% of the cases this would be my first follow up question

Reply:
Thanks! Yes, that's definitely really helpful too.
I'll add kfp deployment, kfp version and sdk version question here too.